README.md

```diff
@@ -7,15 +7,16 @@
  - [x] Tribunal TGIF
  - [ ] Hiking
  - [x] The Social network x1
- - [ ] Brazilian Steakhouse (for a good reason)
+ - [ ] <s>Brazilian Steakhouse (for a good reason)</s>
  - [ ] Kings Island
- - [ ] 16 Bit
+ - [ ] <s>16 Bit</s>
  - [ ] <s>Dueling Piano Bar </s>
  - [ ] Nada
  - [ ] Short Vine Bar Hop - Hit every bar on shortvine from Kroger to Taste of Belgium
  - [x] Engineering Luau
  - [x] Buzzed Bull Creamery
  - [x] Ohio State Fair
  - [ ] Canoeing
- - [ ] Terry's Turf Club
+ - [ ] <s>Terry's Turf Club</s>
  - [ ] Downtown street car barhop
+ - [ ] Napa/Sonoma Meetup
```

Comment:
I'm curious why you wrote Sonoma?

Reply:
It’s nearby and also has wine. Will switch to just Napa.